app/controllers/admins/base_controller.rb

```diff
@@ -1,3 +1,14 @@
 class Admins::BaseController < ApplicationController
+  before_action :check_admin_role
+
   layout "admins"
+
+  private
+
+  def check_admin_role
+    return if current_user.role == User.roles[:admin]
```

Comment:
sử dụng enum luôn e, ngắn mà ơ mây zing thế mà kh dùng à [Doc](https://api.rubyonrails.org/v6.0.3.4/classes/ActiveRecord/Enum.html)
```suggestion
    return if current_user.admin?
```

Reply:
Vâng anh :1st_place_medal: